packages/clerk-js/src/core/clerk.ts

```diff
@@ -271,7 +294,7 @@ export default class Clerk implements ClerkInterface {
     }
 
     //1. setLastActiveSession to passed usersession (add a param)
-    if (typeof document != 'undefined' && document.hasFocus()) {
+    if (session && typeof document != 'undefined' && document.hasFocus()) {
       await this.#touchLastActiveSession(session);
```

Comment:
In this line, we're touching the given session and we're also updating the active organization id with the one that is currently in `session.lastActiveOrganizationId`.

I'm not a huge fan of this. The main reason is that the method's signature hides the above complexity.
If you guys also agree, we should probably change it to something more suitable. 
However, I would prefer if it's done in a separate PR.

Reply:
Should we add a comment that this happens here ?

Reply:
Will do 👍